content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,121 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
+
+   * Kubernetes 1.16 or later
+   * Provider support for dual stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
+   * Kubenet network plugin
+   * Kube-proxy running in mode IPVS
+
+### Enable IPv4/IPv6 dual stack
+
+To enable IPv4/IPv6 dual stack, enable the `IPv6DualStack` [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
+      * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidrs=<IPv4 CIDR>,<IPv6 CIDR>` 
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< note >}}
+An IPv6 CIDR specificed via the `--cluster-cidr` flag larger than /24 will fail
+{{< /note >}}
+
+### Services
+
+If your cluster has IPv4/IPv6 dual-stack networking enabled, you can create {{< glossary_tooltip text="Services" term_id="service" >}} with either an IPv4 or an IPv6 address. You can choose the address family for the Service's cluster IP by setting a field, `.spec.ipFamily`, on that Service.
+You can only set this field when creating a new Service. The default address family for your cluster is the address family of the first service cluster IP range configured via the `--service-cluster-ip-range` flag to the kube-controller- manager.
+
+You can set `.spec.ipFamily` to either:
+
+   * IPv4: api-server will assign an IP from a `service-cluster-ip-range` that is `ipv4`
+   * IPv6: api-server will assign an IP from a `service-cluster-ip-range` that is `ipv6`
+
+The following Service specification includes the `ipFamily` field. Kubernetes will assign an IPv6 address (also known as a "cluster IP") from the configured `service-cluster-ip-range` to this Service.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  ipFamily: IPv6
+  selector:
+    app: MyApp
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 9376
+```
+
+For comparison, the following Service specification will be assigned an IPV4 address (also known as a "cluster IP") from the configured `service-cluster-ip-range` to this Service.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  ipFamily: IPv4
+  selector:
+    app: MyApp
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 9376
+```
+
+#### Type LoadBalancer
+
+On cloud providers which support IPv6 enabled external load balancers, setting the `type` field to `LoadBalancer` provisions a cloud load balancer for your Service.
+
+### Egress Traffic
```

Comment:
That is not true. A CNI can globally reachable `IPv6` (or even `IPv4`). Masquerading is only needed if the user plan to egress on non public/routable `IPv6` addresses.

Reply:
Good point. I will adjust verbiage to represent that